dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -492,6 +495,19 @@ private void handleMap(String repositoryId, String parentFilePath, Version versi
         }
     }
 
+    /**
+     * Sets the type of imports in ParsedInformation based on the import string
+     * @param parsedInformation     A version's version metadata's
+     * @param mapValue              Import string (should be either a local import or an HTTP(s) import
+     */
+    private void setImportsBasedOnMapValue(ParsedInformation parsedInformation, String mapValue) {
+        if (mapValue.startsWith("http")) {
```

Comment:
could this be a problem if some jerk has a folder with a name like `httpStuffIgot/data`/?


https://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/UrlValidator.html might be an option already on our classpath